Gemfile.lock

```diff
@@ -282,8 +281,7 @@ GEM
     naturally (2.2.1)
     netrc (0.11.0)
     no_proxy_fix (0.1.2)
-    nokogiri (1.13.7)
-      mini_portile2 (~> 2.8.0)
+    nokogiri (1.13.7-x86_64-darwin)
```

Comment:
I don't own an M1 (yet, Apple finally started to process my order), I could be talking total nonsense here but does this change mean we'll have to run `arch -x86_64 bundle install` instead of natively without Rosetta?

Reply:
Good catch! I had miss this in the diff, how silly of me given the whole reason I went on my Intel Mac was because of the `nokogiri` failure reported in #385.

I played around with the `bundle` configs in #387 and I seem to be able to run the release task from there, but I do wonder whether the problem is in the task itself, given `bundle install` succeeds from both my machines 🤔

Reply:
This has been a long standing issue with `bundler` and `nokogiri` toggling back and forth between the platform-agnostic version of `nokogiri` and the `x86_64-darwin` one. A change in the `.bundle/config` to add `specific_platform=false` was _supposed_ to solve it for good (see paaHJt-2Gx-p2)… but apparently not always… 😒 :sigh:

So at that point I don't understand why it keeps resolving it to `x86_64-darwin` sometimes, despite `specific_platform` being disabled… 🤷